### PR TITLE
Don't generate tables in the build.rs (do so in advance).

### PR DESCRIPTION
**Pull request by thomcc**
`regex` is a pretty slow dependency to build (well, `regex-syntax` is anyway), and it's pretty unnecessary for this crate to generate the table each time it builds.

I did this by just moving your build.rs to its own crate, which dumps the table to stdout (because it's easy, flexible, but mostly harder to mess up than taking a file argument. Although if you'd rather I do that, I can).

For clarity, `cargo run -p print-table > src/case_folding_data.rs` can update the pre-generated table.

~~Oh, also I updated you to unicode 13.0.0. I did it just because seems like good manners to do so when I'm updating the tables anyway, but I can undo it if you'd prefer.~~

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,10 +6,11 @@ description = "Unicode caseless matching"
 repository = "https://github.com/SimonSapin/rust-caseless"
```
Comment (thomcc) [inline Cargo.toml:6]: (BTW repo url is stale here -- gonna mark this comment as "resolved" since it's irrelevant, but felt like maybe I should mention it)

**Comment by thomcc** [comment]
Ah, there's already a PR for unicode version bump in #14. I'll drop that commit from mine then.

**Comment by crazymerlyn** [comment]
Found this when investigating compile time for RustPython. Looks like SimonSapin has transferred the project to unicode-rs. @Manishearth, does this change seem reasonable to you?

**Review by Manishearth — APPROVED** [review]
This seems fine. Could you also add CI that checks the codegen is up to date?

**Comment by crazymerlyn** [comment]
@Manishearth, thanks for the suggestion. I can't edit this pr, so  created #21 with a workflow. Verified it by manually messing up the file on my own branch: https://github.com/crazymerlyn/rust-caseless/actions/runs/12504550179

**Pull request closed without merging**